assets/js/components/user-input/util/constants.js

```diff
@@ -27,6 +27,14 @@ export const USER_INPUT_QUESTION_GOALS = 'goals';
 export const USER_INPUT_QUESTION_HELP_NEEDED = 'helpNeeded';
 export const USER_INPUT_QUESTION_SEARCH_TERMS = 'searchTerms';
 
+export const QUESTIONS = [
```

Comment:
Let's name this a little more specifically, for consistency with the others but also to make it a bit more clear what it is.

```suggestion
export const USER_INPUT_QUESTIONS_LIST = [
```